Adding ex 11.1

kardaszb opened the pull request.

Commit:
Adding ex 11.1

pawel-javastart commented on src/wh_11_1/EmoloyeeReader.java:
```diff
@@ -0,0 +1,42 @@
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class EmoloyeeReader {
```
literówka

kardaszb replied:
zmienilem

pawel-javastart commented on src/wh_11_1/EmoloyeeReader.java:
```diff
@@ -0,0 +1,42 @@
+        return employes;
+    }
+
+    private static Employe createEmp(String fileLine) {
```
lepiej nie uzywac sktótow i poprostu nazwac metode `createEmployee`

kardaszb replied:
zmienilem :)

pawel-javastart commented on src/wh_11_1/Test.java:
```diff
@@ -0,0 +1,109 @@
+        final String RESULTS_FILE_NAME = "testResults.txt";
+        try {
+            Employe[] employes = EmoloyeeReader.readFromFile(FILE_NAME);
+            for (Employe x : employes) {
```
'x' to zla nazwa. lepiej uzywac pelnej nazwy  `employee`

kardaszb replied:
zmienilem :)

pawel-javastart commented on src/wh_11_1/Test.java:
```diff
@@ -0,0 +1,109 @@
+    }
+
+    private static String minEmployeeSalary(Employe[] employes) {
+        Employe minSalary = employes[0];
```
program nie zadziala dla pustej tablicy

kardaszb replied:
zmienilem:
`        try {
            Employe minSalary = employees[0];
            for (Employe employe : employees) {
                if (employe.getPayment() < minSalary.getPayment())
                    minSalary = employe;
            }
            return "najmniejsza wypłata: " + minSalary.getPayment();
        } catch (ArrayIndexOutOfBoundsException ex) {
            return "Nie da sie policzyc. Pusta tablica";
        }`

pawel-javastart commented on src/wh_11_1/Test.java:
```diff
@@ -0,0 +1,109 @@
+        return "Liczba pracownikow w dziale " + department + ": " + counter;
+    }
+
+    private static void fileWriter(String fileName, String nextLine) {
```
metody dotyczace obslugi plików przerzucilbym do osobnej klasy

kardaszb replied:
przerzucilem

pawel-javastart commented on src/wh_11_1/Test.java:
```diff
@@ -0,0 +1,109 @@
+        }
+    }
+
+    private static String avgEmployeeSalary(Employe[] employes) {
```
metody ktore wyliczaja rozne statystyki przenioslbym do osobnej klasy

pawel-javastart commented on src/wh_11_1/Employe.java:
```diff
@@ -0,0 +1,68 @@
+package wh_11_1;
+
+public class Employe {
```
literówka `Employee`

kardaszb replied:
poprawione

Commit:
EmployeeReader.java:
changed typo in class name

separated from main class:
FileOperations.java
Statistics.java

Exception handled in Statistics.java (min and max value calculation for empty table)
